src/main/java/io/nats/client/impl/ConsumerInfo.java

```diff
@@ -1,152 +1,153 @@
-        }
-
-        @Override
-        public String toString() {
-            return "{" + "consumerSeq=" + consumerSeq + ", streamSeq=" + streamSeq + '}';
-        }
-    }
-
-    private final String stream;
-    private final String name;
-    private final ConsumerConfiguration configuration;
-    private final ZonedDateTime created;
-    private final SequencePair delivered;
-    private final SequencePair ackFloor;
-    private final long numPending;
-    private final long numWaiting;
-    private final long numAckPending;
-    private final long numRedelivered;
-
-    /**
-     * Internal method to generate consumer information.
-     * @param json JSON representing the consumer information.
-     */
-    public ConsumerInfo(String json) {
-        stream = JsonUtils.readString(json, STREAM_NAME_RE);
-        name = JsonUtils.readString(json, NAME_RE);
-        created = JsonUtils.readDate(json, CREATED_RE);
-
-        String jsonObject = JsonUtils.getJSONObject(CONFIG, json);
-        this.configuration = new ConsumerConfiguration(jsonObject);
-
-        jsonObject = JsonUtils.getJSONObject(DELIVERED, json);
-        this.delivered = new SequencePair(jsonObject);
-
-        jsonObject = JsonUtils.getJSONObject(ACK_FLOOR, json);
-        this.ackFloor = new SequencePair(jsonObject);
-
-        numAckPending = JsonUtils.readLong(json, NUM_ACK_PENDING_RE, 0);
-        numRedelivered = JsonUtils.readLong(json, NUM_REDELIVERED_RE, 0);
-        numPending = JsonUtils.readLong(json, NUM_PENDING_RE, 0);
-        numWaiting = JsonUtils.readLong(json, NUM_WAITING_RE, 0);
-    }
-    
-    public ConsumerConfiguration getConsumerConfiguration() {
-        return configuration;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public String getStreamName() {
-        return stream;
-    }
-
-    public ZonedDateTime getCreationTime() {
-        return created;
-    }
-
-    public SequencePair getDelivered() {
-        return delivered;
-    }
-
-    public SequencePair getAckFloor() {
-        return ackFloor;
-    }
-
-    public long getNumPending() {
-        return numPending;
-    }
-
-    public long getNumWaiting() {
-        return numWaiting;
-    }
-
-    public long getNumAckPending() {
-        return numAckPending;
-    }
-
-    public long getRedelivered() {
-        return numRedelivered;
-    }
-
-    @Override
-    public String toString() {
-        return "ConsumerInfo{" +
-                "stream='" + stream + '\'' +
-                ", name='" + name + '\'' +
-                ", numPending=" + numPending +
-                ", numWaiting=" + numWaiting +
-                ", numAckPending=" + numAckPending +
-                ", numRedelivered=" + numRedelivered +
-                ", created=" + created +
-                ", Delivered" + delivered +
-                ", AckFloor" + ackFloor +
-                ", " + configuration +
-                '}';
-    }
-}
+// Copyright 2020 The NATS Authors
```

Comment:
Why the move?   I'd think ConsumerInfo would be a package level class alongside the interfaces...

Reply:
I needed access to things in the impl package.